looper/models.py

```diff
@@ -441,6 +441,8 @@ def __init__(self, config_file, subproject=None,
         self.sheet = None
         self.samples = list()
 
+        self.add_sample_sheet()
```

Comment:
Yeah me either, so I'm ok with it back in too. Just worried about the subclassing thing which seems like a potential pain but like I said probably quite uncommon